css/style.css

```diff
@@ -44,35 +44,113 @@ body {
 }
 
 .bottom-bar {
-  font-family: 'Courier New', Courier, monospace;
   position: absolute;
-  background: #white;
   left: 0;
   bottom: 0;
   height: 50px;
   width: 100%;
-/*  overflow: hidden;*/
 }
 
-.bottom-bar ul {
-  margin: 0% 25% 0 25%;  
+/* helpers/accessibility.css */
+
+.invisible {
+  left: -999px;
+  overflow: hidden;
+  position: absolute;
+  top: -999px;
+}
+
+/* helpers/align.css */
+
+.align {
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
```

Comment:
Check Indentation here please

Reply:
![screenshot_20180512_134200](https://user-images.githubusercontent.com/4529442/39955319-90f6a1fc-55ea-11e8-8ae7-6db7132e0bb4.png)

i did it so that its easy for eyes to find things